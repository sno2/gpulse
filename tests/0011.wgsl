// This declaration hides the predeclared 'min' built-in function.
// Since this declaration is at module-scope, it is in scope over the entire
// source.  The built-in function is no longer accessible.
fn min() -> u32 { return 0; }

const rgba8unorm = 12; // This shadows the predeclared 'rgba8unorm' enumerant.
