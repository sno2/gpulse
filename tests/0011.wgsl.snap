ast.Node{ .fn_decl = ast.FnDecl{ .name = ast.Span{ .start = 206, .end = 209 }, .params = {  }, .ret = ast.Node{ .identifier = ast.Span{ .start = 215, .end = 218 } }, .scope = { ast.Node{ .ret = ast.Ret{ .value = ast.Node{ .number_literal = ast.NumberLiteral{ .span = ast.Span{ .start = 228, .end = 229 }, .kind = ast.NumberLiteral.NumberKind.abstract_int } } } } } } } ast.Node{ .const_decl = ast.ConstDecl{ .name = ast.Span{ .start = 240, .end = 250 }, .typ = null, .value = ast.Node{ .number_literal = ast.NumberLiteral{ .span = ast.Span{ .start = 253, .end = 255 }, .kind = ast.NumberLiteral.NumberKind.abstract_int } } } } 